tests/vptr/CMakeLists.txt

```diff
@@ -25,9 +25,9 @@ if (COMPUTECPP_INTERFACE)
   add_definitions(-DCOMPUTECPP_INTERFACE=${COMPUTECPP_INTERFACE})
   add_executable(runtime runtime.cc)
   set_property(TARGET runtime PROPERTY CXX_STANDARD 11)
```

Comment:
Would you be able to remove this as well? It's not needed, setting the language level to 14 in the base CMakeLists.txt should be enough :)

Reply:
Done!